.github/workflows/label_merge_conflict.yml

```diff
@@ -0,0 +1,21 @@
+name: "Merge Conflict Labeler"
+on:
+  # So that PRs touching the same files as the push are updated
+  push:
+  # So that the `dirtyLabel` is removed if conflicts are resolve
+  # We recommend `pull_request_target` so that github secrets are available.
+  # In `pull_request` we wouldn't be able to change labels of fork PRs
+  pull_request_target:
+    types: [synchronize]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check if prs are dirty
+        uses: eps1lon/actions-label-merge-conflict@v3
+        with:
+          dirtyLabel: "merge-conflicts"
+          removeOnDirtyLabel: "ready"
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
+          commentOnDirty: "This pull request has conflicts, please resolve those before we can evaluate the pull request."
```

Comment:
qq: who will the author of comments?

Reply:
it's https://github.com/vllm-ascend-ci IMO. @hfadzxy please doube check. Thanks. does `GITHUB_TOKEN ` is provided by https://github.com/vllm-ascend-ci ?